schema/library.raml

```diff
@@ -373,7 +372,7 @@ types:
       isMergeKey: boolean
       isMultiValued: boolean
       imputeZero: boolean
-      hasStudyDependentVocabulary?: boolean
+      hasStudySpecificVocabulary?: boolean
```

Comment:
whats happened here? this isnt right..

Reply:
i ran `make raml-gen-code` which made this change. Is the original name the correct one? I can remove this change from the PR